README.md

```diff
@@ -138,10 +138,10 @@ To flatten one iterable:
 ```java
 new Joined<>(
   new Mapped<>(
-    iter -> new IterableOf<>(
-      new CollectionOf<>(iter).toArray(new Integer[]{})
-    ),
-    new IterableOf<>(new IterableOf<>(1, 2, 3, 4, 5, 6))
+      iter -> new IterableOf<>(
```

Comment:
@fanifieiev as I can see, there are 2 spaces indents in this code fragment. I suggest not to change it to 4.

Reply:
@iakunin Fixed